lib/utils/BaseReporter.js

```diff
@@ -283,6 +343,51 @@ class BaseReporter extends events.EventEmitter {
         }
     }
 
+    getSpecStats (obj) {
+        if (!this.stats.runner[obj.cid]) {
```

Comment:
Hmm. I'm not very familiar with `Object.assign`. Could you show me how to use it in this case?